EIPS/eip-3675.md

```diff
@@ -0,0 +1,244 @@
+
+### Deprecating `extraData`
+
+The **`extraData`** field is deprecated in favour of the corresponding **`graffiti`** field of the beacon block.
+
+### Changing block validity rules
+
+The rule set enforcing the PoW seal validity is replaced with the corresponding PoS rules along with the consensus upgrade as the rationale behind this change.
+
+An additional rule validating a set of deprecated block fields is required by the block format changes introduced by this specification.
+
+### Removing block rewards
+
+Existing rewards for producing and sealing blocks are deprecated along with the PoW mechanism. The new PoS consensus becomes both responsible for sealing blocks and for issuing block rewards once this specification enters into effect.
+
+### Supplanting fork choice rule
+
+The fork choice rule of the PoW mechanism becomes completely irrelevant after the upgrade and is replaced with the corresponding rule of the new PoS consensus mechanism.
+
+### Removing block gossip
+
+After the upgrade of the consensus mechanism only the beacon chain network will have enough information to validate a block. Thus, block gossip provided by the `eth` network protocol will become unsafe and is deprecated in favour of the block gossip existing in the beacon chain network. 
+
+It is recommended for the client software to not propagate descendants of any terminal PoW block to reduce the load on processing the P2P component and stop operating in the environment with unknown security properties.
+
+## Test Cases
+*TBD*
+
+## Reference Implementation
+*TBD*
+
+
+## Backwards Compatibility
+
+This EIP introduces backward incompatibilities in block validity, block rewards and fork choice rule.
+
+The design of the consensus upgrade specified by this document does not introduce backward incompatibilities for existing applications and services built on top of Ethereum except for those that are described in the [EVM](#EVM) section below or heavily depends on the PoW consensus in any other way.
+
+
+### EVM
+
+Although this EIP does not introduce any explicit changes to the EVM there are a couple of places where it may affect the logic of existing smart contracts.
+
+#### DIFFICULTY
+
+`DIFFICULTY` operation will always return `0` after this EIP takes effect and deprecates the **`difficulty`** field by replacing it with `0` constant.
+
+*Note:* Altering the `DIFFICULTY` semantics to return randomness accumulated by the beacon chain is under consideration but will be introduced in a separate EIP.
+
+#### BLOCKHASH
+
+Pseudo-random numbers obtained as the output of `BLOCKHASH` operation become more insecure after this EIP takes effect and the PoW mechanism (which decreases the malleability of block hashes) gets supplanted by PoS.
+
+
+## Security Considerations
+
+### Beacon chain
+
+See Security Considerations section of [EIP-2982](/EIPS/eip-2982#security-considerations).
```

Comment:
```suggestion
See Security Considerations section of [EIP-2982](/EIPS/eip-2982).
```

Reply:
is it possible to preserve the anchors?

Reply:
I would leave them as well. What is the reason to remove them?